openjdk.build/include/top.xml

```diff
@@ -191,12 +191,17 @@ limitations under the License.
 	</target>
 
 	<!-- Make sure mauve.jar is built. -->
-	<target name="run-mauve-configure" depends="fetch-prestaged-mauve-jar" unless="isAutomatedBuild">
+	
+	<target name="run-mauve-configure" unless="prestaged-prereqs-available">
```

Comment:
Instead of making target 'check-if-prestaged-prereqs-available' and `fetch-prestaged-mauve-jar` as a global one and scatter around I would suggest do like <target name="run-mauve-configure" depends="fetch-prestaged-mauve-jar" unless="prestaged-prereqs-available">, 

Hence <target name="fetch-prestaged-mauve-jar" depends=""check-if-prestaged-prereqs-available"  if="prestaged-prereqs-available">

Reply:
Or could be even more straightforward as 
`if jar is there
     cp
else 
    run mauve configure
endif
`

Reply:
You are right about using if-else. However, the systemtest Ant scripts have been written to avoid use of if-then-else directives. I just try to keep that coding standard going :).

Reply:
```Instead of making target 'check-if-prestaged-prereqs-available' and fetch-prestaged-mauve-jar as a global one and scatter around I would suggest do like
<target name="run-mauve-configure" depends="fetch-prestaged-mauve-jar" unless="prestaged-prereqs-available">``` - yes, that could be done!. I've updated the PR, please refresh.

Reply:
Tested this change by locally running the system tests ; and for Jenkins scenario, in this Grinder: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/436/console